discovery-infra/install_cluster.py

```diff
@@ -112,12 +114,10 @@ def run_install_flow(client, cluster_id, kubeconfig_path, pull_secret):
     )
 
 
-def download_logs_from_all_hosts(client, cluster_id):
-    output_folder = f'build/{cluster_id}'
-    utils.recreate_folder(output_folder)
+def download_logs_from_all_hosts(client, cluster_id, output_folder):
     hosts = client.get_cluster_hosts(cluster_id=cluster_id)
     for host in hosts:
-        output_file = os.path.join(output_folder, f'{host["id"]}_logs.tar.gz')
+        output_file = os.path.join(output_folder, f'host_{host["id"]}.tar.gz')
         waiting.wait(
             lambda: client.download_host_logs(cluster_id=cluster_id,
                                               host_id=host["id"],
```

Comment:
Cluster logs already contain host logs - not needed IMO
@tsorya

Reply:
right missed it

Reply:
but then we need to validate in installation flow that all relevant logs are here.